david_goodedit/src/App.js

```diff
@@ -4,14 +4,32 @@ import InputArea from './components/InputArea'
 import Home from './components/Home'
 import Tutorial from './components/Tutorial'
 
-function App() {
-  return (
-    <div className='App'>
-        <Home />
-        <InputArea />
+class App extends React.Component {
+  constructor(props) {
+    super(props)
+    this.state = {
+      light: true
+    }
+    this.aziz = this.aziz.bind(this)
+  }
+
+  aziz() {
+    this.setState({ light: !this.state.light })
+    if (this.state.light === true) {
+      document.documentElement.setAttribute('data-theme', 'light')
+    } else {
+      document.documentElement.setAttribute('data-theme', 'dark')
+    }
+  }
+  render() {
+    return (
+      <div className='darkApp'>
+        <Home aziz={this.aziz} light={this.state.light} />
+        <InputArea light={this.state.light} />
```

Comment:
try:

      aziz() {
        this.setState({ light: !this.state.light })
      }
      render() {
        return (
          <div className='darkApp' data-theme={this.state.light ? 'light' : 'dark'}>